### PR TITLE
Addresses #386, fix misc pool tsv files and updates project_singlefamilydetached masonry walls

### PR DESCRIPTION
**Pull request by joseph-robertson**
Addresses #386.

## Pull Request Description

[description here]

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] PAT project measures/outputs have been updated
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing (green) on circleci
- [ ] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: project_multifamily_beta/housing_characteristics/Misc Pool Heater.tsv [hunk 1]
```diff
@@ -1,2 +1,3 @@
-Option=Electric	Option=Gas	Option=None
-0.004000	0.014000	0.982000
+Dependency=Misc Pool	Option=Electric	Option=Gas	Option=None
+None	0.000000	0.000000	1.000000
+Has Pool	0.053333	0.186667	0.760000
```
Comment (ejhw) [inline project_multifamily_beta/housing_characteristics/Misc Pool Heater.tsv]: Where are these values from?

File: resources/options_lookup.tsv [hunk 2]
```diff
@@ -1825,26 +1825,26 @@ Misc Hot Tub Spa	None	ResidentialMiscLargeUncommonLoads	has_hot_tub_heater_elec=
...
+Misc Pool Schedule	None	ResidentialMiscLargeUncommonLoads	"pool_weekday_sch=0.003, 0.003, 0.003, 0.004, 0.008, 0.015, 0.026, 0.044, 0.084, 0.121, 0.127, 0.121, 0.120, 0.090, 0.075, 0.061, 0.037, 0.023, 0.013, 0.008, 0.004, 0.003, 0.003, 0.003"	"pool_weekend_sch=0.003, 0.003, 0.003, 0.004, 0.008, 0.015, 0.026, 0.044, 0.084, 0.121, 0.127, 0.121, 0.120, 0.090, 0.075, 0.061, 0.037, 0.023, 0.013, 0.008, 0.004, 0.003, 0.003, 0.003"	"pool_monthly_sch=1.154, 1.161, 1.013, 1.010, 1.013, 0.888, 0.883, 0.883, 0.888, 0.978, 0.974, 1.154"																																										
+Misc Pool Schedule	National Average	ResidentialMiscLargeUncommonLoads	"pool_weekday_sch=0.003, 0.003, 0.003, 0.004, 0.008, 0.015, 0.026, 0.044, 0.084, 0.121, 0.127, 0.121, 0.120, 0.090, 0.075, 0.061, 0.037, 0.023, 0.013, 0.008, 0.004, 0.003, 0.003, 0.003"	"pool_weekend_sch=0.003, 0.003, 0.003, 0.004, 0.008, 0.015, 0.026, 0.044, 0.084, 0.121, 0.127, 0.121, 0.120, 0.090, 0.075, 0.061, 0.037, 0.023, 0.013, 0.008, 0.004, 0.003, 0.003, 0.003"	"pool_monthly_sch=1.154, 1.161, 1.013, 1.010, 1.013, 0.888, 0.883, 0.883, 0.888, 0.978, 0.974, 1.154"
```
Comment (ejhw) [inline resources/options_lookup.tsv]: Is this schedule used for Pool Heaters too?

File: project_multifamily_beta/housing_characteristics/Misc Pool Heater.tsv (hunk 1 above)
Comment (joseph-robertson) [reply project_multifamily_beta/housing_characteristics/Misc Pool Heater.tsv]: I took these from the values for National Average in the options lookup.

* 7.5% of houses have pools (pool_pump_mult=0.075)
* 1.8% of houses have pool heaters (pool_heater_elec_mult=0.004 + pool_heater_gas_mult=0.014)

So 1.8/7.5=24% of homes with pools have pool heaters, and 100-24=76% of homes with pools do not have pool heaters.

**Comment by ejhw** [comment]
I'm good with this, but I'd like @JLReyna to take a quick look to be aware of the low sample sizes in some cases. We might consider having fewer regions and/or combining with RECS 2015 to get more samples.

**Pull request merged**